clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentReadGroup.scala

```diff
@@ -2,15 +2,16 @@ package org.broadinstitute.clio.server.dataaccess.elasticsearch
 
 import java.time.OffsetDateTime
 
-case class DocumentReadGroup(analysisType: Option[String],
+case class DocumentReadGroup(flowcellBarcode: String,
+                             lane: Int,
+                             libraryName: String,
+                             location: String,
```

Comment:
I gave it a shot in https://github.com/broadinstitute/clio/commit/b3a19ecf40d56ccca6b35600eac978536fde790e
It's not too clunky, but I welcome suggestions.